scripts/helper/fullpath.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+fullpath ()
+{
+  fullPath=$(cd $1 && pwd -P)
+  echo $fullPath
+  cd $OLDPWD	
```

Comment:
I do not believe the final `cd` is needed here and maybe has an undesired side effect?   Since line 5 is ran in a subshell the current directory is not changed